firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Bytes.txt

```diff
@@ -0,0 +1,59 @@
+Scenario:The Util get_byte function correctly gets bytes from the given position
+   GIVEN("The util get_byte function")
+       WHEN("the get_byte function is called on 0x86 ")
+          THEN(""The value at the least significant byte position is "0x0a'")
+   GIVEN("The util get_byte function " )
+       WHEN("the get_byte function is called on 0x860a")  
+       THEN(""The value at the least significant byte position is "0x0a")
+       THEN("The value at the most signficant postiion is 0x86")
```

Comment:
I would suggest specifying byte indices, rather than talking about MSB and LSB (because above uint16_t, there are bytes in between MSB and LSB). The `WHEN`s also need to specify the template parameters, as mentioned in our test patterns document on Notion: e.g. `WHEN('the function is called with byte index 0 and input type uint16_t as the template parameters and with 0x86 as the input value')` and `THEN('the returned value is 0x86')`, vs. `WHEN('the function is called with byte index 1 and input type uint16_t as the template parameters and with 0x86 as the input value')` and `THEN('the returned value is 0x00')`. Since the byte index specified as a function parameter, it should be specified in the `WHEN` and not the `THEN`.